.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,6 +1,6 @@
 ---
-name: Bug report
-about: Create a report to help us improve
+name: 'Problem encountered while running DeepVariant'
```

Comment:
Not sure if these quotes will show up in the final title. If they do, we can remove as a followup.

Reply:
Good question, let's see!

Reply:
Looks good, no quotes! :)